src/iconsManifest/supportedExtensions.ts

```diff
@@ -3753,6 +3753,21 @@ export const extensions: IFileCollection = {
       extensions: ['mli'],
       format: FileFormat.svg,
     },
+    {
+      icon: 'dune',
+      extensions: ['dune', 'dune-project', 'dune-workspace'],
+      filenamesGlob: ['dune', 'dune-project', 'dune-workspace'],
```

Comment:
There's no need for this. We can keep this array empty.

Reply:
The files are named literally just `dune`, `dune-project` and `dune-workspace` without a "classical" extension. Is that how you'd do that still?

Reply:
If yes, done

Reply:
yes, that's correct. The `filenamesGlob` it's just used in case the filenames were `dune.dune`, `dune.dune-project`, `dune.dune-workspace`...